import-scripts/pipelines/mycrontab

```diff
@@ -0,0 +1,42 @@
+MAILTO=cbioportal-pipelines@cbio.mskcc.org
```

Comment:
Note to self: although mycrontab is not a called script, this file perhaps should be also "deployed" to the upper level scripts directory .. to sit beside automation_environment. (as a reference to maintainers)